clang/include/clang/Driver/Driver.h

```diff
@@ -254,6 +254,9 @@ class Driver {
   /// stored in it, and will clean them up when torn down.
   mutable llvm::StringMap<std::unique_ptr<ToolChain>> ToolChains;
 
+  /// Number of parallel jobs.
+  unsigned NumParallelJobs;
+
```

Comment:
This might not be required for now.

Reply:
Agreed, don't think it's necessary for functionality